lib/vechain/lib/single-node-stack.ts

```diff
@@ -0,0 +1,148 @@
+import * as cdk from "aws-cdk-lib";
+import * as cdkConstructs from "constructs";
+import * as ec2 from "aws-cdk-lib/aws-ec2";
+import * as iam from "aws-cdk-lib/aws-iam";
+import * as s3Assets from "aws-cdk-lib/aws-s3-assets";
+import * as path from "path";
+import * as fs from "fs";
+import * as cw from "aws-cdk-lib/aws-cloudwatch";
+import * as nag from "cdk-nag";
+import { SingleNodeConstruct } from "../../constructs/single-node";
+import * as constants from "../../constructs/constants";
+import { VETNodeSecurityGroupConstruct } from "./constructs/vet-node-security-group";
+import { SingleNodeCWDashboardJSON } from "./constructs/node-cw-dashboard";
+import { VetNetwork, VetNodeType } from "./config/node-config.interface";
+import { DataVolumeConfig } from "../../constructs/config.interface";
+
+
+
+export interface VetSingleNodeStackProps extends cdk.StackProps {   
+    vetNodeType: VetNodeType;
+    syncFromPublicSnapshot: boolean;
+    instanceType: ec2.InstanceType;
+    instanceCpuType: ec2.AmazonLinuxCpuType;
+    dataVolume: DataVolumeConfig;
+    network: VetNetwork;
+    vetContainerImage: string;
+    nodeHealthcheckTolerance: number;
+    nodeHealthcheckVersion: string;
+    instanceRole: iam.Role;
+}
+
+export class VETSingleNodeStack extends cdk.Stack {
+    constructor(scope: cdkConstructs.Construct, id: string, props: VetSingleNodeStackProps) {
+        super(scope, id, props);
+
+        // Setting up necessary environment variables
+        const REGION = cdk.Stack.of(this).region;
+        const STACK_NAME = cdk.Stack.of(this).stackName;
+        const STACK_ID = cdk.Stack.of(this).stackId;
+        const availabilityZones = cdk.Stack.of(this).availabilityZones;
+        const chosenAvailabilityZone = availabilityZones.slice(0, 1)[0];
+
+        // Getting our config from initialization properties
+        const {
+            instanceType,
+            instanceCpuType,
+            syncFromPublicSnapshot,
+            vetNodeType,
+            dataVolume,
+            network,
+            vetContainerImage,
+            nodeHealthcheckTolerance,
+            nodeHealthcheckVersion,
+            instanceRole
+        } = props;
+
+        // Using default VPC
+        const vpc = ec2.Vpc.fromLookup(this, "vpc", { isDefault: true });
+
+        // Setting up the security group for the node from XRP-specific construct
+        const instanceSG = new VETNodeSecurityGroupConstruct(this, "security-group", {
+            vpc: vpc,
+            vetNodeType
+        });
+
+        // Making our scripts and configis from the local "assets" directory available for instance to download
```

Comment:
```suggestion
        // Making our scripts and configs from the local "assets" directory available for instance to download
```
Use a spellchecker for typos, I have one recommended in [the onboarding checklist](https://vechain.atlassian.net/wiki/spaces/NodeOps/pages/1211465729/Onboarding+checklist)